### ISSUE TITLE
Fail to build on FreeBSD due to incorrect include of wait.h

### ISSUE THREAD

**Issue by sexybiggetje** [HaxeFoundation/ocamllibs#15]
Hi,

Not an experienced c dev, so instead of opening a pull request I would like to see this this way, because i can't foresee trouble. Building fails because of the following in extc/process_stubs.c. 

```
#       ifndef __APPLE__
#               include <wait.h>
#       endif
```

It should be patched to reference sys/wait.h, both the [linux man page](http://linux.die.net/man/2/wait) as the [ports man page](https://www.freebsd.org/cgi/man.cgi?query=wait&sektion=2) show this path.

Patching this fixes the complete build for me.

**Comment by ppenzin** [comment]
Proposing a fix; successfully built it on FreeBSD 10.1 and Linux (Ubuntu 14.04)

**Comment by sexybiggetje** [comment]
Builds on 10.1-RELEASE and 10.2-RELEASE for me. Thanks!
